src/lib/snapp.ts

```diff
@@ -268,6 +272,83 @@ function isProof(typ: any) {
   return false; // TODO
 }
 
+type Statement = { transaction: Field; atParty: Field };
+
+function toStatement(
```

Comment:
It would be nice if you included a short snippet from your PR description that describes what a `Statement` is and why it's used here.

Info I'm referencing:
... 
constructing the public input -- called the statement in the code. The statement consists of certain hashes of the transaction and of the proving Party which is constructed during method execution. 
...

Some of this information would be really helpful to have instead of hunting through PRs later on

Reply:
good point - added a comment: https://github.com/o1-labs/snarkyjs/pull/123/commits/65bbe45dfbe8599b6322278d804366f979536493